src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label class="form_field">
+          Surname :
+          <input
+            name="Surname"
+            type="text"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="35"
+          >
+        </label>
+        <label class="form_field">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Fix it everywhere 
```suggestion
        </label>
        
        <label class="form_field">
```